node_modules/oae-core/activity/js/activity.js

```diff
@@ -122,8 +122,9 @@ define(['jquery', 'underscore', 'oae.core', 'activityadapter'], function($, _, o
             infinityScroll = $('.oae-list', $rootel).infiniteScroll(url, {
                 'limit': 10
             }, '#activity-items-template', {
-                'postProcessor': processActivities,
-                'emptyListProcessor': handleEmptyResultList
+                'ariaLive': 'off',  // Notifications will alert screen readers
```

Comment:
Not all new activities will cause a notification to be shown. We only show the notifications for new "notifications". That means a screenreader would currently not see any other new activities.